libs/java/server_common/src/main/java/com/yahoo/athenz/common/server/util/config/ConfigEntry.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright 2017 Yahoo Inc.
```

Comment:
Please fix Copyright, the first line should be "Copyright The Athenz Authors"

Reply:
Fixed